### ISSUE TITLE
`Map.groupBy` and `Object.groupBy`

### ISSUE THREAD

**Issue by fregante** [dustinspecker/eslint-plugin-no-use-extend-native#152]
The plugin is warning on:

```js
const users = Map.groupBy(found, element => element.textContent.trim());
// Avoid using extended native objects (no-use-extend-native/no-use-extend-native) XO no-use-extend-native/no-use-extend-native

```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/groupBy

Is this a bug or a misconfiguration? I'm using it on a TypeScript file

**Comment by dustinspecker** [comment]
Thank you for raising this issue. This looks like xo is using an older version of eslint-plugin-no-use-extend-native: https://github.com/xojs/xo/blob/1877df631c64229d628440cb913f0ad2e58e346e/package.json#L73

The current version of this plugin (0.7.2) allows Map/Object.groupBy: https://github.com/dustinspecker/eslint-plugin-no-use-extend-native/commit/a6d54db48147c7a9df053521aa33fd80915a0f7f
